client/extensions/woocommerce/app/order/order-details-table.js

```diff
@@ -80,7 +86,7 @@ class OrderDetailsTable extends Component {
 				<TableItem isHeader className="order__detail-item-product">{ translate( 'Product' ) }</TableItem>
 				<TableItem isHeader className="order__detail-item-cost">{ translate( 'Cost' ) }</TableItem>
 				<TableItem isHeader className="order__detail-item-quantity">{ translate( 'Quantity' ) }</TableItem>
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Quantity:' )` *ES Score: 26*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).